### PR TITLE
[AFC] Implemented Wand Of Orcus

### PR DESCRIPTION
**Pull request by zeffirojoe**
#7978

### CODE REVIEW

File: Mage.Sets/src/mage/cards/w/WandOfOrcus.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        Ability deathTouchAbility = new AttacksOrBlocksEnchantedTriggeredAbility(Zone.BATTLEFIELD,
+                new GainAbilityAttachedEffect(DeathtouchAbility.getInstance(), AttachmentType.EQUIPMENT,
+                        Duration.EndOfTurn));
+        deathTouchAbility.addEffect(
+                new GainAbilityControlledEffect(DeathtouchAbility.getInstance(), Duration.EndOfTurn, filter));
+        this.addAbility(deathTouchAbility);
+
+        // Whenever equipped creature deals combat damage to a player, create that many
+        // 2/2 black Zombie creature tokens.
+        this.addAbility(new WandOfOrcusZombieAbility());
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/w/WandOfOrcus.java:60]: you can use `DealsDamageToAPlayerAttachedTriggeredAbility` instead of creating a custom class

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/w/WandOfOrcus.java (hunk 1 above)
Comment (zeffirojoe) [reply Mage.Sets/src/mage/cards/w/WandOfOrcus.java:60]: Is there an example of this where the damage value is used? That was the reason I created the custom class in the first place. Backfire is the only one i see ?

Comment (theelk801) [reply Mage.Sets/src/mage/cards/w/WandOfOrcus.java:60]: it’s saved to the effects in the `checkTrigger` method

Comment (zeffirojoe) [reply Mage.Sets/src/mage/cards/w/WandOfOrcus.java]: makes sense, thanks for the feedback. I committed a re-attempt

File: Mage.Sets/src/mage/cards/w/WandOfOrcus.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    @Override
+    public WandOfOrcusZombieEffect copy() {
+        return new WandOfOrcusZombieEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Integer damage = (Integer) this.getValue("damage");
+        if (damage != null) {
+            return (new CreateTokenEffect(new ZombieToken(), damage).apply(game, source));
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/w/WandOfOrcus.java:94]: one last thing, here you don't need to use `CreateTokenEffect`, you can just use `new ZombieToken().putOntoBattlefield`

Comment (zeffirojoe) [reply Mage.Sets/src/mage/cards/w/WandOfOrcus.java]: Got it, made the change.

**Pull request merged**